nemo/collections/nlp/parts/mixins/nlp_adapter_mixins.py

```diff
@@ -178,9 +179,10 @@ def _check_and_add_peft_cfg(self, peft_cfg):
                 for layer in layers:
                     if layer.layer_number in (layer_selection or list(range(1, self.cfg.num_layers + 1))):
                         for name, module in layer.named_modules():
-                            self._check_and_add_adapter(
-                                name, module, adapter_name, adapter_cfg, name_key_to_mcore_mixins
-                            )
+                            if not isinstance(module, IdentityOp):
+                                self._check_and_add_adapter(
```

Comment:
is it better to add this check inside `self._check_and_add_adapter`?

Reply:
Done